### PR TITLE
Build.sh

### PR DESCRIPTION
**Pull request by anmol2302**
1: Changed SCHEMA FOR CERTIFICATE ADD in registry

### CODE REVIEW

File: all-actors/src/main/java/org/sunbird/CertVars.java [hunk 1]
```diff
@@ -14,30 +14,26 @@
     private  static final String SERVICE_BASE_URL=getPropsFromEnvs(CERT_SERVICE_BASE_URL);
     private static final String DOWNLOAD_URI= "/v1/user/certs/download";
```
Comment (manzarul) [inline all-actors/src/main/java/org/sunbird/CertVars.java:15]: use enum for Uri

File: sb-utils/src/main/java/org.sunbird/JsonKeys.java [hunk 2]
```diff
@@ -51,7 +51,15 @@
     public static final String TYPE = "type";
     public static final String ACCOUNT = "account";
     public static final String key = "key";
-
+    public static final String USER_NAME="userName";
+    public static final String RECIPIENT_TYPE="recipientType";
+    public static final String REASON="reason";
+    public static final String COURSE_COMPLETION="course-completion";
```
Comment (manzarul) [inline sb-utils/src/main/java/org.sunbird/JsonKeys.java:57]: course-completion need to be camel case same for course-performance

Comment (anmol2302) [reply sb-utils/src/main/java/org.sunbird/JsonKeys.java:57]: @manzarul not required becuase same key need to be matched from request.

File: all-actors/src/main/java/org/sunbird/CertVars.java (hunk 1 above)
Comment (anmol2302) [reply all-actors/src/main/java/org/sunbird/CertVars.java:15]: Will change when add VERIFY API

Comment (manzarul) [reply all-actors/src/main/java/org/sunbird/CertVars.java:15]: ok

**Pull request merged**